Projects/TEAM-1/Turbal Y/Task 1_5/Задача 1/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Задача_1
```

Comment:
Please don't use Cyrillic for namespaces, class names, etc.
Cyrillic is allowed only for comments.